godotproject/project.godot

```diff
@@ -72,8 +72,6 @@ tongue={
 2d_physics/layer_2="player"
 2d_physics/layer_3="enemy"
 2d_physics/layer_4="tongueable"
-2d_physics/layer_5="spikes"
-2d_physics/layer_6="star"
```

Comment:
Why did you delete my physics layers? Does the star collection still work like this?

Reply:
mmm, I'm not sure how these got deleted. Definitely was not intentional, we can add it back in